cartridge

```diff
@@ -1953,12 +1953,29 @@ local cmd_start = {
             --foreground    Do not daemonize
 
         Default options can be overriden in ./.cartridge.yml or ~/.cartridge.yml.
-    ]]):format(self_name):gsub('(\n?)' .. (' '):rep(8), '%1'),
+
+        When APP_NAME is not provided it is parsed from ./*.rockspec filename or
+        can be set as `app` in config file.
+
+        When INSTANCE_NAME is not provided it reads `cfg` file and starts all
+        defined instances.
+    ]=]):format(self_name):gsub('(\n?)' .. (' '):rep(8), '%1'),
 }
 
+-- Fetches app_name from .rockspec file.
+-- Extracted from cartridge.argparse, but searches for rockspec only in current
+-- directory.
+local function get_app_name()
+    local rockspecs = fio.glob('*-scm-1.rockspec')
```

Comment:
What if there is no such file in the directory. How it will behave? I think it should throw user an error (or usage)

Reply:
It can be set in different ways. Final value is checked on line 2070

Reply:
Continued [here](https://github.com/tarantool/cartridge-cli/pull/9/files#r321174638)